packages/client/mutations/AcceptRequestToJoinDomainMutation.ts

```diff
@@ -25,47 +22,14 @@ const mutation = graphql`
     }
   }
 `
-type Handlers = {
-  onSuccess?: () => void
-  onError?: () => void
-}
-const useAcceptRequestToJoinDomainMutation = () => {
-  const [commit, submitting] = useMutation<TAcceptRequestToJoinDomainMutation>(mutation)
-  const atmosphere = useAtmosphere()
-  const execute = (
-    config: UseMutationConfig<TAcceptRequestToJoinDomainMutation>,
-    handlers?: Handlers
-  ) => {
-    const {variables} = config
-    return commit({
-      onCompleted: (res) => {
-        const error = res.acceptRequestToJoinDomain.error
 
-        if (!error) {
-          atmosphere.eventEmitter.emit('addSnackbar', {
-            message: '🎉 Success! User added',
-            autoDismiss: 5,
-            key: 'acceptRequestToJoinDomainSuccess'
-          })
-          SendClientSegmentEventMutation(atmosphere, 'Join Request Reviewed', {
-            action: 'accept',
-            teamIds: variables.teamIds
-          })
-          handlers?.onSuccess && handlers.onSuccess()
-        } else {
-          atmosphere.eventEmitter.emit('addSnackbar', {
-            message: error.message,
-            autoDismiss: 5,
-            key: 'acceptRequestToJoinDomainError'
-          })
-          handlers?.onError && handlers.onError()
-        }
-      },
-      // allow components to override default handlers
-      ...config
-    })
-  }
-  return [execute, submitting] as const
+const useAcceptRequestToJoinDomainMutation = () => {
+  return useGenericMutation(
+    mutation,
+    'acceptRequestToJoinDomain',
```

Comment:
we can remove this as a param & use `getRequest(mutation).operation.selections[0].name`